server/s3_object_metadata.cc

```diff
@@ -1686,19 +1688,15 @@ bool S3ObjectMetadata::check_bucket_replication_policy(
       }
     }
 
-    if (check_replication_filter_match(rule_object, object_tags_map)) {
-      get_rule_configuration_from_replication_policy(
-          rule_object, new_dest_bucket_name, new_delete_marker_rep_status,
-          new_rule_priority);
-      check_and_update_destination_bucket_map(new_dest_bucket_name,
-                                              new_delete_marker_rep_status,
-                                              new_rule_priority);
+    if (!check_replication_filter_match(rule_object, object_tags_map)) continue;
```

Comment:
Put the "continue" on next line.

Reply:
It is done after I use clang-format -style.
If I forcefully put it on second line then verification/cla-signed error will be shown.

Reply:
OK. But this is something new to me.